qfieldcloud_sdk/sdk.py

```diff
@@ -482,6 +484,8 @@ def upload_file(
         Returns:
             The response object from the upload request.
         """
+        is_valid_filepath(str(local_filename))
```

Comment:
```suggestion
        # if the filepath is invalid, it will throw a new error `pathvalidate.ValidationError`
        is_valid_filepath(str(local_filename))
```

And I guess we need to reflect this in the documentation string of the function too.